runtime/bcverify/staticverify.c

```diff
@@ -297,6 +298,13 @@ checkBytecodeStructure (J9CfrClassFile * classfile, UDATA methodIndex, UDATA len
 					errorDataIndex = index;
 					goto _verifyError;
 				}
+			} else if (index >= cpCount - 1) {
```

Comment:
Please add parentheses for cpCount - 1. e.g.  index >= (cpCount - 1).
The rest of code changes looks good to me.

Reply:
fixed